src/components/ProfileDetails/ProfileDetails.js

```diff
@@ -0,0 +1,69 @@
+import { useQuery} from "@apollo/client";
+import { GET_ALL_USERS, GET_ALL_USER_INFO } from '../../queries/queries';
+
+/*
+users {
+    id
+    userName
+    firstName
+    lastName 
+    email
+    phoneNumber
+    birthday
+    posts{
+      content
+			}
+  }
+
+*/
+
+const ProfileDetails = () => {
```

Comment:
This setup looks like it should be easy enough to translate for other querying functionality